src/Compatibility/Microsoft.DotNet.PackageValidation/Resources.resx

```diff
@@ -118,16 +118,16 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="CompatibleFrameworkInPackageValidatorHeader" xml:space="preserve">
-    <value>Api surface area should be compatible among compatible frameworks.</value>
+    <value>{0} assembly api surface area should be compatible with {1} assembly surface area so we can compile against {0} and run on {1} .framework.</value>
   </data>
   <data name="CompatibleTfmValidatorHeader" xml:space="preserve">
-    <value>The runtime assets should be compatible with the compile time assets for all target frameworks and rids.</value>
+    <value>The compile time assemblies public api surface area should be compatible with the runtime assemblies for all target frameworks and rids. Please add the following apis to the runtime assemblies.</value>
   </data>
   <data name="InvalidPackagePath" xml:space="preserve">
     <value>Package Path {0} is invalid. Please Check the path .</value>
   </data>
   <data name="ApiCompatibilityHeader" xml:space="preserve">
-    <value>There are API Compatibility errors in between {0} (left) and {1} (right).</value>
+    <value>Api Compatibility errors  between {0} (left) and {1} (right) are as follows:</value>
```

Comment:
```suggestion
    <value>API Compatibility errors between {0} (left) and {1} (right):</value>
```

Reply:
I don't think we should be exposing the terms "left" and "right" to end users.  Instead we should be giving them descriptive identifiers, such as:
compile asset, runtime asset
framework X asset, compatible framework Y asset
etc

Reply:
Yes I totally agree with you. This is temporary while I add support for API Compat to have custom naming for `left` and `right`. However, since at the moment it is hardcoded to `left` and `right` in the API Compat diagnostic messages, it made sense to add it here so that it is clearer what's `left` and what's `right`.

Reply:
I see, makes sense.

Reply:
I created: https://github.com/dotnet/sdk/issues/17495 to track this.